### PR TITLE
wip: CreateElementTool. Fix not being able to pick decorations after changing files.

### PR DESCRIPTION
**Pull request by bbastings**
Still working on porting sketch tools from Gist, wanted to push the fix for pickable decorations asap.

### CODE REVIEW

File: core/frontend/src/AccuDraw.ts [hunk 1]
```diff
@@ -2916,6 +2920,7 @@ export class AccuDraw {
       (current.view.iModel === previous.view.iModel))
       return;
 
+    this._acsPickId = undefined;
```
Comment (pmconne) [inline core/frontend/src/AccuDraw.ts:2923]: Is this one line the only thing required to fix picking decorations after changing files?

**Review by bsteinbk — APPROVED** [review]

File: core/frontend/src/AccuDraw.ts (hunk 1 above)
Comment (bbastings) [reply core/frontend/src/AccuDraw.ts:2923]: Yes.

Comment (pmconne) [reply core/frontend/src/AccuDraw.ts:2923]: Does the WIP stuff break anything? Are you expecting the WIP stuff to be reviewed too? Do you think maybe the one-line fix should be a separate PR?

Comment (bbastings) [reply core/frontend/src/AccuDraw.ts:2923]: The WIP stuff is fully functional and replaces code that was removed from ui-test-app...I just plan to make additional changes based on the set of sketching tools in Gist.

The DynamicGraphicsProvider and how it's used in CreateLineStringTool is worth a look.

**Review by pmconne — APPROVED** [review]

**Pull request merged**